ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/workspace/WorkspaceServiceClientImpl.java

```diff
@@ -290,7 +296,11 @@ public void makeCall(AsyncCallback<WorkspaceDto> callback) {
     private void deleteCommand(@NotNull final String wsId,
                                @NotNull final String commandName,
                                @NotNull AsyncCallback<WorkspaceDto> callback) {
-        final String url = baseHttpUrl + '/' + wsId + "/command/" + commandName;
+        String url = null;
+        try {
+            url = baseHttpUrl + '/' + wsId + "/command/" + URLEncoder.encode(commandName, "UTF-8");
+        } catch (UnsupportedEncodingException ignored) {
+        }
```

Comment:
I'm not sure silent catch is good as after that url is null and it will create exceptions